src/components/Router.js

```diff
@@ -48,6 +49,6 @@ export default class Router extends Component {
       return h2('No matching path.')
     }
     const el = h(handler.component, Nav.getHandlerProps(handler, pathname))
-    return handler.public ? el : h(AuthContainer, [el])
+    return h(FooterContainer, [handler.public ? el : h(AuthContainer, [el])])
```

Comment:
Note: I wanted to render this outside the Router, but couldn't because the footer needs to generate links. Because we're using hot loading, the route definitions are effectively stateful, so timing matters. Just wanted to call this out as a cost we're paying for hot loading.